.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: rust
+
+branches:
+  only:
+  - master
+
+os:
+  - linux
+
+matrix:
+  include:
+    - rust: nightly
+      after_script:
+          # Send a notification to the Dusk build Status Telegram channel once the CI build completes
+        - bash <(curl -s https://raw.githubusercontent.com/dusk-network/tools/master/bash/telegram_ci_notifications.sh)
+
```

Comment:
We should also publish the docs. But it's fine atm

Reply:
I copied one of the `.travis.yml` we have in our repo, what would a more recent one?
Or, should we already move out from `travis` also here and move to github actions? (In that case, I will file a follow up issue)

Reply:
I don't have any problem staying with travis. Specially because the builds & tests here should not take too long since there's not any ZK stuff going on.

The block you're missing for the docs upload is the following:
```yaml
# Upload docs
  - |
      if [[ "$TRAVIS_OS_NAME" == "linux" && "$TRAVIS_RUST_VERSION" == "nightly" && "$TRAVIS_PULL_REQUEST" = "false" && "$TRAVIS_BRANCH" == "master" ]]; then
        cargo rustdoc --features nightly -- --html-in-header katex-header.html &&
        echo "<meta http-equiv=refresh content=0;url=dusk_plonk/index.html>" > target/doc/index.html &&
        git clone https://github.com/davisp/ghp-import.git &&
        ./ghp-import/ghp_import.py -n -p -f -m "Documentation upload" -r https://"$GH_TOKEN"@github.com/"$TRAVIS_REPO_SLUG.git" target/doc &&
        echo "Uploaded documentation"
      fi
```

i usually have it inside an `after_success:` block

Reply:
Replace `dusk_plonk` by the name of the crate and it should work without any problems.

Reply:
I might just use the Makefile for that, in this case.